### PR TITLE
Updated comment section for CustomFieldSource to explain how to norma…

### PR DESCRIPTION
**Pull request by prashkh**
…lize the results properly

### CODE REVIEW

**Review by tomflexcompute — APPROVED** [review]
Thanks Prash. There seems to be an extra space between `grid` and `is` in line 742.

File: tidy3d/components/source.py [hunk 1]
```diff
@@ -737,6 +737,12 @@ class CustomFieldSource(FieldSource, PlanarSource):
         the directionality of the source will likely not be perfect, even if both the ``E`` and ``H`` fields are
         provided. An empty normalizing run may be needed to accurately normalize results.
 
+        To create this empty simulation it is recommended that users create a simulation with no structures but just a flux
+        monitor (``tidy3D.FluxMonitor``) next to the custom source, ensuring that the flux monitor is at least one grid cell
+        away from the source. Moreover, for accurate normalization, users must ensure that the same grid  is used to run
+        the original simulation as well as the empty simulation. The total flux calculated at the flux monitor of the empty
```
Comment (momchil-flex) [inline tidy3d/components/source.py:763]: I think adding a comment about how to achieve the same grid when using auto grid will also be useful. Namely what we did yesterday: define the same structures you used to have in your original Simulation in the GridSpec `override_structures` of the empty sim.

**Review by tylerflex — APPROVED** [review]

File: tidy3d/components/source.py (hunk 1 above)
Comment (prashkh) [reply tidy3d/components/source.py:763]: Thanks for the feedback! Added a short explanation on how to create empty simulation with the grid from original simulation.

**Review by momchil-flex — APPROVED** [review]
Looks good, could you just squash the three commits into 1 and then force push?

**Comment by prashkh** [comment]
> Looks good, could you just squash the three commits into 1 and then force push?

@momchil-flex  I squashed the commits and force pushed it. Let me know if it is all good.

**Comment by momchil-flex** [comment]
For some reason it is still 3 commits @prashkh - but I'll just do it myself.

**Comment by prashkh** [comment]
@momchil-flex thanks so much! not sure what happened but I thought I squashed and force pushed it

**Pull request merged**